packages/cerebral-provider-http/src/utils.js

```diff
@@ -64,3 +64,20 @@ export function convertObjectWithTemplates (obj, context) {
     return convertedObject
   }, {})
 }
+
+export function parseHeaders (rawHeaders) {
```

Comment:
Ah, they do? Sorry, I thought there was a native method for grabbing them. Okay, but looks good :)